pfrl/experiments/evaluator.py

```diff
@@ -363,14 +366,28 @@ def eval_performance(
             max_episode_len=max_episode_len,
             logger=logger,
         )
-    stats = dict(
-        episodes=len(scores),
-        mean=statistics.mean(scores),
-        median=statistics.median(scores),
-        stdev=statistics.stdev(scores) if len(scores) >= 2 else 0.0,
-        max=np.max(scores),
-        min=np.min(scores),
-    )
+    if isinstance(scores, tuple):
+        reward_scores = scores[0]
+        success_rate = scores[1]
+        stats = dict(
+            episodes=len(reward_scores),
+            mean=statistics.mean(reward_scores),
+            median=statistics.median(reward_scores),
+            stdev=statistics.stdev(reward_scores) if len(reward_scores) >= 2 else 0.0,
+            max=np.max(reward_scores),
+            min=np.min(reward_scores),
+            success_rate=success_rate,
+        )
+
+    else:
+        stats = dict(
+            episodes=len(scores),
+            mean=statistics.mean(scores),
+            median=statistics.median(scores),
+            stdev=statistics.stdev(scores) if len(scores) >= 2 else 0.0,
+            max=np.max(scores),
+            min=np.min(scores),
+        )
```

Comment:
Another option is to add switch to `batch_run_evaluation_episodes` and `run_evaluation_episodes` to return scores AND success_rate. That way the code might be a bit cleaner.